easybuild/easyconfigs/g/GMP/GMP-6.2.0-GCCcore-9.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'GMP'
+version = '6.2.0'
+
+homepage = 'https://gmplib.org/'
+description = """
+ GMP is a free library for arbitrary precision arithmetic, operating on signed
+ integers, rational numbers, and floating point numbers.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'precise': True, 'pic': True}
+
+source_urls = ['https://ftp.gnu.org/gnu/%(namelower)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['f51c99cb114deb21a60075ffb494c1a210eb9d7cb729ed042ddb7de9534451ea']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('binutils', '2.34'),
+]
+
+# enable C++ interface
+configopts = '--enable-cxx'
+
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ['lib/libgmp.%s' % SHLIB_EXT, 'include/gmp.h'],
```

Since we are specifically enabling `cxx` I would check for its libs and headers as well:
```suggestion
    'files': ['lib/lib%s.%s' % (l, e) for l in ['gmp', 'gmpxx'] for e in [SHLIB_EXT, 'a']] +
             ['include/gmp.h', 'include/gmpxx.h'],    
```